### ISSUE TITLE
[REQ] Add Fail2ban

### ISSUE THREAD

**Issue by IgorekG** [Zer0CoolX/guacamole-install-rhel#123]
**Confirmation of Process**
1. You have read the [How to report issues](https://github.com/Zer0CoolX/guacamole-install-rhel/wiki/How-to-Report-Issues-(Bugs,-Feature-Request-and-Help)) wiki page? Y
2. You have read the wiki, especially the To Do: [Planned and Desired Features](https://github.com/Zer0CoolX/guacamole-install-rhel/wiki/To-Do:-Planned-and-Desired-Features) page? Y
3. You have read other requests to ensure this feature has not already been requested? Y

**In a sentence or 2, describe briefly what the idea is**
Adding fail2ban. Complete!

**Is your feature request related to a problem?**
N

**Describe the idea/feature you'd like**
I want that guacamole can block brutforcing login and password

**Any links, documentation or resource that would help implement this feature**
https://www.wilbursecurity.com/2016/08/fail2ban_guacamole/

**Additional context**

**Comment by Zer0CoolX** [comment]
Thank you for the request, but you must have overlooked it in my plans: https://github.com/Zer0CoolX/guacamole-install-rhel/projects/2. As of now I am feature freezing while I work on adapting the script to RHEL/CentOS 8.

**Comment by IgorekG** [comment]
I didn't see your plans, sorry. But I looked over your opend issues and didn's find open fail2ban request becouse it closed. Why?
I thought that the request should be closed only when it is completed.
Thanks

**Comment by Zer0CoolX** [comment]
> I thought that the request should be closed only when it is completed.

Not always. If you read the other f2ban request, I was waiting to hear back from the OP and they never replied back. I closed it as I had:

- Expected a response from the OP and never got one
- Added f2ban to my plans
- Having the issue open provided no further value and only added clutter to the open and active issues.

Since then things have changed. I have limited time to devote to this script and have to decide how best to use that limited time. In many cases that means doing something at the expense of not doing something else. Right now, getting a script working on CentOS/RHEL 8.x > just about anything else.

Also, being an open source project, people are welcome to contribute code to it. If someone else can code a feature, test it, verify it works and contributes the code its one less thing I have to consider using my time on.

I hope this clarifies how I have handled the matter.
